src/parser/jobs/mnk/BuffUptime.js

```diff
@@ -0,0 +1,161 @@
+import React, {Fragment} from 'react'
+
+import {ActionLink} from 'components/ui/DbLink'
+import ACTIONS, {getAction} from 'data/ACTIONS'
+import STATUSES from 'data/STATUSES'
+import Module from 'parser/core/Module'
+import {Rule, Requirement} from 'parser/core/modules/Checklist'
+import {Suggestion, SEVERITY} from 'parser/core/modules/Suggestions'
+
+const GCD_CYCLE_LENGTH = 5
```

Comment:
Shouldn't this actually be 6?

Reply:
It's zero indexed. If we initialise it at 1, then it should be 6. Maybe for clarity that's a good idea though, but then it looks weird when we have "1 GCD since DK/TS" when they reset it.

Reply:
Oops, yes.